persistence/src/commonMain/db_user/com/wire/kalium/persistence/Messages.sq

```diff
@@ -128,6 +128,16 @@ CREATE TABLE MessageMissedCallContent (
       PRIMARY KEY (message_id, conversation_id)
 );
 
+CREATE TABLE MessageConversationChangedContent (
+      message_id TEXT NOT NULL,
+      conversation_id TEXT AS QualifiedIDEntity NOT NULL,
+
```

Comment:
i guess we need to store the id of the user who changed the name

Reply:
This is stored in the `Messages` parent table, so this is only related for this log kind of system messages :D